File: pyton/impares.py
print("Digite dois numeros: ")
numero1 = int(input())
numero2 = int(input())

if numero1 > numero2:
    troca = numero1
    numero1 = numero2
    numero2 = troca
    
soma = 0
for i in range(numero1+1, numero2):
    if i % 2 != 0:
        soma = soma + i
    
print(f"SOMA DOS IMPARES = {soma}")

File: c/estrutura_para/impares.c
#include <stdio.h>

int main(void)
{
    int x, y, soma, troca;
    
    printf("Digite dois numeros:\n");
    scanf("%i %i", &x, &y);
    
    if (x > y)
    {
        troca = x;
        x = y;
        y = troca;
    }
    
    soma = 0;
    
    for (int i = x+1; i < y; i++)
    {
        if (i % 2 != 0)
        {
            soma = soma + i;
        }
    }
    
    printf("SOMA DOS IMPARES = %i\n", soma);
    
}
